authorization/auth_test.go

```diff
@@ -21,6 +22,11 @@ func TestMain(m *testing.M) {
 		log.Fatalln("failed to parse args")
 	}
 
+	err = iam.ParseArgs()
```

Comment:
How come we have two ParseArgs() funcs to call?

Reply:
One is for authentication, the other is for environments

Reply:
Fixed